### ISSUE TITLE
Docker buildx component missing in amazonlinux2-aarch64-standard:3.0

### ISSUE THREAD

**Issue by alexgmin** [aws/aws-codebuild-docker-images#640]
Same issue as https://github.com/aws/aws-codebuild-docker-images/issues/625 but for al2, as mentioned here. https://github.com/aws/aws-codebuild-docker-images/issues/625#issuecomment-1546702657

Note that even though https://github.com/aws/aws-codebuild-docker-images/issues/625 is closed, the fix is not working, as indicated in https://github.com/aws/aws-codebuild-docker-images/issues/625#issuecomment-1558485411

**Describe the bug**

Refers to [Docker release note](https://docs.docker.com/engine/release-notes/23.0/):

From Docker Engine version 23.0.0, Buildx is distributed in a separate package: docker-buildx-plugin. In earlier versions, Buildx was included in the docker-ce-cli package. When you upgrade to this version of Docker Engine, make sure you update all packages.

This will result in Dockerfile stated BUILDKIT=1 to fail.

**To Reproduce** 
1. Add a new build project and set the image to amazonlinux2-aarch64-standard:3.0
2. In buildspec.yml, use these lines:
```yaml
- export DOCKER_BUILDKIT=1
- docker build .
```
4. Trigger build
5. See error:

```
ERROR: BuildKit is enabled but the buildx component is missing or broken.
       Install the buildx component to build images with BuildKit:
       https://docs.docker.com/go/buildx/
```

**Expected behavior** 
This will success in amazonlinux2-aarch64-standard:3.0

**Logs** 
N/A

**Platform (please complete the following information):** 
N/A

**Additional context** 
N/A

**Comment by BwL1289** [comment]
is there an update on this?

**Comment by SupSkiFun** [comment]
Still occurring on 06SEP2023.
Image	aws/codebuild/amazonlinux2-aarch64-standard:3.0
Environment type	ARM

**Without DOCKER_BUILDKIT set you get a warning:**

DEPRECATED: The legacy builder is deprecated and will be removed in a future release.
            Install the buildx component to build images with BuildKit:
            https://docs.docker.com/go/buildx/

**Updating buildspec.yml with**:

env:
  variables:
    DOCKER_BUILDKIT: 1

**Causes a failure**:

 ERROR: BuildKit is enabled but the buildx component is missing or broken.
       Install the buildx component to build images with BuildKit:
       https://docs.docker.com/go/buildx/

Would be nice for this to work without needing to make a symlink or install a package.  Thanks.

**Comment by mhoeper** [comment]
Same here, AWS fixed this in aws-codebuild-docker-images/al2/x86_64/**standard**, but forgot to fix in aws-codebuild-docker-images/al2/**aarch64**
/standard...

I hope this to be addressed soon, as this should be easy one.

**Comment by Taikono-Himazin** [comment]
Please let me know the progress on this.
I think this is a big problem and a simple solution, but the fix is too slow.

**Comment by abhishek-parative** [comment]
Still affected by this issue

**Comment by vib8m** [comment]
Still facing this. Any updates?

**Comment by Dylan-AWS** [comment]
This was addressed in #697
